examples/kotlin-pingpong/src/main/proto/cloudstate/entity_key.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Lightbend Inc.
```

Comment:
are copies of the Cloudstate protocol protobuf files need within the example? or could this be get from a dependency?

Reply:
Things that you haven't been able to solve yet, today you can't compile all the files directly present. I will open issues on the build topics

Reply:
It should be possible to depend on the zip-archives and extract them as a part of the build?

Reply:
> It should be possible to depend on the zip-archives and extract them as a part of the build?

I will check this

Reply:
@sleipnir Any news on this?

Reply:
@viktorklang I opened a problem to address this [issue](https://github.com/cloudstateio/kotlin-support/issues/5) later. Continue to evaluate the issue.